.github/actions/cl2-modules/netpol/modules/metrics.yaml

```diff
@@ -14,7 +15,7 @@ steps:
         metricName: NetPol Policy Implementation Delay
         metricVersion: v1
         unit: s
-        enableViolations: true
+        enableViolations: {{$ENABLE_VIOLATIONS}}
```

Comment:
unrelated to your changes but curious: does this prevent failure if a query threshold is exceeded? can't find mention of it in CL2 docs

Reply:
> unrelated to your changes but curious: does this prevent failure if a query threshold is exceeded? can't find mention of it in CL2 docs

yup, correct. Tldr we have defaults set for 100 nodes, but when running 1k nodes, it obviously fails but is still good for measuring memory/cpu etc.